src/components/application_manager/test/commands/mobile/perform_audio_pass_thru_test.cc

```diff
@@ -346,7 +346,7 @@ TEST_F(PerformAudioPassThruRequestTest, OnTimeout_GENERIC_ERROR) {
   command->onTimeOut();
 
   ResultCommandExpectations(
-      msg_mobile_response, NULL, am::mobile_api::Result::GENERIC_ERROR, false);
+      msg_mobile_response, "", am::mobile_api::Result::GENERIC_ERROR, false);
```

Comment:
@VVeremjova fixed in https://github.com/smartdevicelink/sdl_core/pull/1368/commits/e1e998ffc82c12f956a5d10777d9061aa13daae2